tests/fixtures_utils.py

```diff
@@ -177,6 +177,14 @@ def _get_data_metrics(well_fixture):
     return peak_detection.data_metrics(peak_and_valley_indices, filtered_data)
 
 
+def _get_unrounded_data_metrics(well_fixture):
+    pipeline, peak_and_valley_indices = well_fixture
+    filtered_data = pipeline.get_noise_filtered_gmr()
+    return peak_detection.data_metrics(
+        peak_and_valley_indices, filtered_data, rounded=False
+    )
+
+
 def assert_percent_diff(actual, expected, threshold=0.0006):
```

Comment:
note for future: now that we know about the `pytest.approx` method, I think this could probably be removed, and anywhere it is being used replaced with `approx`